### PR TITLE
feat: [ListV2] Connect modal styling to functionality

### PR DESCRIPTION
**Pull request by cbachmeier**
Connected new v2 modal to listing logic. Still needs error states and success screen.
connectedListingV2

### CODE REVIEW

**Review by JackShort — APPROVED** [review]
noice

File: src/nft/components/icons.tsx [hunk 1]
```diff
@@ -317,7 +317,7 @@ export const ApprovedCheckmarkIcon = (props: SVGProps) => (
   <svg width="28" height="28" viewBox="0 0 28 28" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
     <path
       d="M13.7464 21.1101C13.3792 21.6796 12.8372 22 12.2076 22C11.5607 22 11.0536 21.733 10.5814 21.1101L6.45464 15.9844C6.15738 15.6107 6 15.2191 6 14.792C6 13.9021 6.66448 13.208 7.52131 13.208C8.0459 13.208 8.46557 13.4038 8.88525 13.9555L12.1377 18.1735L19.0448 6.88988C19.412 6.28476 19.8842 6 20.4087 6C21.2306 6 22 6.58732 22 7.4594C22 7.86874 21.7902 8.29588 21.5628 8.66963L13.7464 21.1101Z"
-      fill="currentColor"
+      fill={props.fill ? props.fill : 'currentColor'}
```
[the remark this answers is not in the document]
Comment (cbachmeier) [reply src/nft/components/icons.tsx:320]: Great suggestion, ticket added https://uniswaplabs.atlassian.net/browse/NFT-1067

File: src/nft/components/profile/list/ListPage.tsx [hunk 2]
```diff
@@ -124,28 +129,58 @@ const ListingButtonWrapper = styled.div`
...
+      setCollectionsRequiringApproval,
+    }),
+    shallow
+  )
 
   const totalEthListingValue = useMemo(() => getTotalEthValue(sellAssets), [sellAssets])
   const anyListingsMissingPrice = useMemo(() => !!listings.find((listing) => !listing.price), [listings])
-  const [ethPriceInUSD, setEthPriceInUSD] = useState(0)
   const [showListModal, toggleShowListModal] = useReducer((s) => !s, false)
+  const [selectedMarkets, setSelectedMarkets] = useState([ListingMarkets[0]]) // default marketplace: x2y2
```
[the remark this answers is not in the document]
Comment (cbachmeier) [reply src/nft/components/profile/list/ListPage.tsx:173]: Added to this ticket https://uniswaplabs.atlassian.net/browse/NFT-91

File: src/nft/components/icons.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (cbachmeier) [reply src/nft/components/icons.tsx]: Changed here to use Feather

**Pull request merged**